pages/guides/live/place-a-bet.mdx

```diff
@@ -1,7 +1,234 @@
-import { Callout } from 'components'
+import { Callout, Image } from 'components'
 
 # Place a bet
 
+Placing a bet on live games in Azuro is a multistep process that can be described by the next diagram:
+
+<Image
+  src="/images/live/place-a-bet.png"
```

Comment:
it's better to avoid presenting the whole view of how our feature works. Better to keep it blackboxed. Hide everything that happens on backend side under "Azuro Live Engine"